src/deferred/exceptionHook.js

```diff
@@ -0,0 +1,21 @@
+define( [
+	"../core",
+	"../deferred"
+], function( jQuery ) {
+
+// These usually indicate a programmer mistake during development,
+// warn about them ASAP rather than swallowing them by default.
+var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;
+
+jQuery.Deferred.exceptionHook = function( error ) {
+
+	// Support: IE9
+	// Console exists when dev tools are open, which can happen at any time
+	var warn = window.console && window.console.warn;
```

Comment:
This needs to have the context bound to `console` doesn't it?  Otherwise it throws `Uncaught TypeError: Illegal invocation` in Chrome (probably others too)